src/makeCalculator.js

```diff
@@ -36,8 +36,49 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      const operation = calculator.result + number;
+
+      return operation;
+    },
+
+    subtract(number) {
+      const operation = calculator.result - number;
```

Comment:
Instead of creating a new variable 'operation', directly update the 'result' property